.github/workflows/reusable-docs.yml

```diff
@@ -40,12 +40,14 @@ jobs:
       uses: Ana06/get-changed-files@v2.2.0
       with:
         filter: "Doc/**"
+        format: csv  # works for paths with spaces
     - name: 'Build changed files in nit-picky mode'
       if: github.event_name == 'pull_request'
       continue-on-error: true
       run: |
+        set -Eeuo pipefail
```

Comment:
I think we can skip this? 

```suggestion
```

Reply:
We we sure that it's okay to let arbitrary failures go unnoticed here?

Reply:
What sort of arbitrary failures?

I think it's okay, this step is for adding annotations about warnings, but not failing the build if a problem is found.

Reply:
@hugovk

For example, a pipe fail right before the last command would be ignored or referencing non-existing vars due to future changes with typos etc.

I do understand that it's okay for this step to fail. But I'd expect it to show up red in the log when it does, not green. If it's green, people wouldn't know to expand it and look inside if something misbehaves.

`continue-on-error: true` will make sure it doesn't fail the job in either case — the only difference is when something behaves weirdly in the middle of the script and it's hidden by a green success checkmark.